media/admin/com_virtuemart/models/zasilkovna_orders.php

```diff
@@ -738,4 +740,77 @@ function getOrderLineDetails($orderId, $orderLineId) {
         }
     }
 
+    /**
+     * Shows errors in module administration
+     */
+    public function raiseErrors()
+    {
+        if (is_array($this->errors)) {
+            foreach ($this->errors as $error) {
+                JError::raiseWarning(600, $error);
```

Comment:
Proč 600 ?

Reply:
Tak mam dat 601? Stejne je to v jinym modelu.

Reply:
Zkus prosím zjistit proč se tam dává 600

Reply:
z anotace te metody:
	 * @param   string  $code  The application-internal error code for this error
	 * 
takze by to mel byt spis string, ale vsude mozne v joomle se tam bezne posila cislo.
Nikde jide se to nase 600 nepouziva.
Nakonec se to code preda do exceptiony jako exception->code